components/patient-access-rules/ui/src/main/resources/PhenoTips/PatientAccessRightsManagement.xml

```diff
@@ -972,6 +979,16 @@
         }
       }
       return result;
+    },
+
+    _removeCollaboratorRow : function (e) {
+      (e.findElement &amp;&amp; event.findElement('tr') || e).remove();
+      this.collaboratorRemoved = true;
+    },
+
+    _removeCollaborator : function (id) {
```

Comment:
It's actually for https://github.com/phenotips/phenomecentral.org/pull/92/files#diff-34db1051f5ac1ee33b3cf5f1a4a3dd29R201